src/centos/stream9/helix/amd64/Dockerfile

```diff
@@ -6,10 +6,22 @@ RUN dnf install --setopt tsflags=nodocs --refresh -y \
     && \
     dnf config-manager --set-enabled crb \
     && \
-    dnf install --setopt tsflags=nodocs -y \
+    dnf install --setopt tsflags=nodocs -y --allowerasing \
+        autoconf \
+        automake \
+        curl \
+        file \
         gcc \
+        gcc-c++ \
```

Comment:
I'm wondering why we need gcc/g++ for Helix run. We may have them in other images but AFAIK Helix only needs Python. And we should not need `devel` packages - that is for building...